rewrite-java/src/main/java/org/openrewrite/java/format/WrapMethodInvocationArguments.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.format;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.service.SourcePositionService;
+import org.openrewrite.java.style.WrappingAndBracesStyle;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaSourceFile;
+import org.openrewrite.java.tree.Space;
+import org.openrewrite.style.LineWrapSetting;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class WrapMethodInvocationArguments<P> extends JavaIsoVisitor<P> {
+
+    WrappingAndBracesStyle style;
+
+    @Override
+    public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, P ctx) {
+        J.MethodInvocation m = super.visitMethodInvocation(method, ctx);
+
+        try {
+            // styles are parent loaded, so the getters may or may not be present and they may or may not return null
+            if (style != null && style.getMethodCallArguments() != null && style.getMethodCallArguments().getWrap() != LineWrapSetting.DoNotWrap) {
```

Comment:
If we're doing `null` checks here, should we annotate the `style` field as nullable?
I can see it's a pattern copied from `WrapMethodDeclarationParameters`; same applies there.